src/parser/jobs/smn/modules/GeneralCDDowntime.js

```diff
@@ -0,0 +1,23 @@
+import ACTIONS from 'data/ACTIONS'
+import CooldownDowntime from 'parser/core/modules/CooldownDowntime'
+
+export default class GeneralCDDowntime extends CooldownDowntime {
+	//Time that Jump deems ok for a OGCD to be down : ^)
```

Comment:
jump does smn stuff? :woah:

Reply:
I copy/pasted that file from RDM and adjusted the skill list.  I just left the note as what little "where this number came from" as was there.